package.json

```diff
@@ -25,8 +25,8 @@
     "format:prettier": "prettier \"**/*.js\" \"**/*.ts\" \"**/*.mjs\"  --write --ignore-path .gitignore",
     "lint": "npm run lint:eslint && npm run lint:prettier",
     "format": "npm run format:eslint && npm run format:prettier",
-    "storybook": "npm run bootstrap && npm run storybook:analyze && start-storybook -p 6006 -s ./.storybook/images",
```

Comment:
@iOvergaard what is the reason to remove "-s ./.storybook/images"?

Reply:
@nielslyngsoe It is deprecated and instead you set up staticDirs in main.js

Reply:
Perfect, thanks for the explanation.